noxfile.py

```diff
@@ -202,13 +198,11 @@ def github_actions_default_tests(session: nox.Session) -> None:
 
 @nox.session(
     python=[
-        "3.7",
         "3.8",
         "3.9",
         "3.10",
         "3.11",
         "3.12",
-        "pypy3.7",
         "pypy3.8",
```

Comment:
This is also EOL fwiw. Last released was on 2022-12-29.

Reply:
Ahh, I thought I got PyPy everywhere. It's a bit tricky to grep since we support targeting older Pythons.

Technically, pypy3.8 is EoL too. pypy2.7, pypy3.9, and pypy3.10 are the only supported ones. But I think matching CPython is fine.

Reply:
Ah, you mean the line below, pypy3.8. Yes, but we've always just matched CPython. It's not extra effort to support (in fact, you can't split them up in requires-python IIRC)